src/components/constants.ts

```diff
@@ -35,3 +35,6 @@ export const FOCUSABLE_SELECTOR = [
 	'audio[controls]',
 	'video[controls]'
 ].join(', ');
+
+export const ADDON_ID = 'storybook/dark-mode';
+export const TOOL_ID = `${ADDON_ID}/tool`;
```

Comment:
Move these constants inside the addon itself since they are strictly related to it (the register file)